src/brpc/controller.cpp

```diff
@@ -1027,6 +1037,43 @@ void Controller::SubmitSpan() {
     _span = NULL;
 }
 
+void* Controller::HandleIdleProgressiveReader(void* arg) {
+    auto* cntl = static_cast<Controller*>(arg);
+    const uint64_t CHECK_INTERVAL_US = 1000000UL;
```

Comment:
It seems that the check interval is 1s, but the resolution of progressive_read_timeout is 1ms